repo/init.go

```diff
@@ -206,6 +206,20 @@ func addConfigExtensions(repoRoot string) error {
 				MediumFeeDefault: 10,
 				LowFeeDefault:    5,
 			},
+			ETH: &schema.CoinConfig{
```

Comment:
We will probably need a migration for this since it wont be in existing configs. We can probably hold off on that though and bundle it with the other migration we need to do related to the insight servers.

Reply:
@cpacia I don't know what that scope is and would like it to be tracked. Can you please make an issue for the required migrations so we don't lose it?